packages/mui-codemod/src/v5.0.0/base-hook-imports.js

```diff
@@ -0,0 +1,113 @@
+const capitalize = (str) => {
+  return str.charAt(0).toUpperCase() + str.slice(1);
+};
+
+const list = [
```

Comment:
```suggestion
const movedExports = [
```

Let's give this list a more descriptive name.